arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -165,5 +165,52 @@ public async Task GetVertexCollectionsAsync_ShouldThrow_WhenGraphDoesNotExist()
             Assert.Equal(HttpStatusCode.NotFound, apiError.Code);
             Assert.Equal(1924, apiError.ErrorNum);
         }
+
+        [Fact]
+        public async Task PostGraphAsync_ShouldSucceed()
+        {
+            var graphName = nameof(PostGraphAsync_ShouldSucceed) + "_clx";
```

Comment:
Note: `_graph` would be a better suffix, it's a graph name, not a collection name. Otherwise no suffix works also I think.